tools/graph_generation/grapher.r

```diff
@@ -0,0 +1,63 @@
+install.packages("ggplot2")
+library("ggplot2")
+library("plyr")
+
+setwd("C:\\Users\\tirpi\\Downloads")
+ue4_stats <- read.csv("ue4_stats.csv", row.names = NULL)
+acl_stats <- read.csv("acl_stats.csv", row.names = NULL)
```

Comment:
they are being read from the working dir which is the only required argument of the script